src/cleanvision/imagelab.py

```diff
@@ -661,11 +685,15 @@ def load(
         with open(object_file, "rb") as f:
             imagelab: TImagelab = pickle.load(f)
 
-        if data_path is not None:
-            filepaths = get_filepaths(data_path)
-            if set(filepaths) != set(imagelab._filepaths):
-                raise ValueError(
-                    "Absolute path of image(s) has changed in the dataset. Cannot load Imagelab."
-                )
+        # todo: use hash for validating
+        # if data_path is not None:
+        #     filepaths = get_filepaths(data_path)
+        #     if set(filepaths) != set(imagelab._filepaths):
+        #         raise ValueError(
+        #             "Absolute path of image(s) has changed in the dataset. Cannot load Imagelab."
+        #         )
         print("Successfully loaded Imagelab")
         return imagelab
+
+    def get_index_to_path_mapping(self) -> Dict[int, str]:
+        return self._dataset.get_index_to_path_mapping()
```

Comment:
also strange that this core functionality could exist without being covered by an existing unit test, maybe codecov is not tracking execution properly.

Reply:
It's not being used in the code anywhere only in tutorial and there was no unit test for it.